src/sql/parts/jobManagement/views/jobsView.component.ts

```diff
@@ -25,15 +26,15 @@ import { CommonServiceInterface } from 'sql/services/common/commonServiceInterfa
 import * as sqlops from 'sqlops';
 import * as vscode from 'vscode';
 import * as nls from 'vs/nls';
-import { IGridDataSet } from 'sql/parts/grid/common/interfaces';
-import { FieldType, IObservableCollection, CollectionChange, SlickGrid } from 'angular2-slickgrid';
 import { Table } from 'sql/base/browser/ui/table/table';
 import { attachTableStyler } from 'sql/common/theme/styler';
 import { JobHistoryComponent } from './jobHistory.component';
 import { AgentViewComponent } from '../agent/agentView.component';
 import { RowDetailView } from 'sql/base/browser/ui/table/plugins/rowdetailview';
 import { JobCacheObject } from 'sql/parts/jobManagement/common/jobManagementService';
 import { AgentJobUtilities } from '../common/agentJobUtilities';
+import { HeaderFilter } from '../../../base/browser/ui/table/plugins/headerFilter.plugin';
```

Comment:
absolute path

Reply:
@anthonydresser I'll clean everything up and upload screenshots. It's currently WIP but it's almost done.